### PR TITLE
[#3936] web ci retry testClickCatalogLink at most 5 times

### PR DESCRIPTION
**Pull request by LauraXia123**
### What changes were proposed in this pull request?
retry testClickCatalogLink at most 5 times

### Why are the changes needed?
api data inconsistencies with test cast by the instability response

Fix: #3936

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
manual test

### CODE REVIEW

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java [hunk 1]
```diff
@@ -57,6 +59,9 @@ public class CatalogsPageTest extends AbstractWebIT {
   protected static String mysqlUri = "jdbc:mysql://127.0.0.1";
   protected static String postgresqlUri = "jdbc:postgresql://127.0.0.1";
 
+  private static final long MAX_WAIT_IN_SECONDS = 5;
+  private static final long WAIT_INTERVAL_IN_SECONDS = 1;
```
Comment (yuqi1129) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java:63]: I would suggest reducing the value of the wait interval, maybe 500ms or so is more suitable.

Comment (LauraXia123) [reply integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java:63]: 'WAIT_INTERVAL_IN_SECONDS' is refer to other it test for this value.

**Comment by LauraXia123** [comment]
rerun ci over 10 times
image

**Review by yuqi1129 — APPROVED** [review]
LGTM

**Pull request merged**